### PR TITLE
Change react description and root url

### PR DESCRIPTION
**Pull request by ph1p**
Changed description wording (mentioned in slack)
And I have updated the root URLs, as France, for example, has done: https://github.com/reactjs/fr.reactjs.org/blob/master/gatsby-config.js

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -8,10 +8,10 @@
 
 module.exports = {
   siteMetadata: {
-    title: 'React: Eine JavaScript Bibliothek für das Bauen von Benutzeroberflächen',
-    siteUrl: 'https://reactjs.org',
+    title: 'React: Eine JavaScript Bibliothek für das Erstellen von Benutzeroberflächen',
+    siteUrl: 'https://de.reactjs.org',
     rssFeedTitle: 'React',
-    rssFeedDescription: 'Eine JavaScript Bibliothek für das Bauen von Benutzeroberflächen',
+    rssFeedDescription: 'Eine JavaScript Bibliothek für das Erstellen von Benutzeroberflächen',
```
Comment (ChrisB9) [inline gatsby-config.js]: sounds way better! :fireworks: 

optional addition:
```suggestion
    rssFeedDescription: 'Eine JavaScript Bibliothek zum Erstellen von Benutzeroberflächen',
```

Comment (ph1p) [reply gatsby-config.js]: right! sounds more colloquial.

File: gatsby-config.js [hunk 2]
```diff
@@ -8,10 +8,10 @@
 
 module.exports = {
   siteMetadata: {
-    title: 'React: Eine JavaScript Bibliothek für das Bauen von Benutzeroberflächen',
-    siteUrl: 'https://reactjs.org',
+    title: 'React: Eine JavaScript Bibliothek für das Erstellen von Benutzeroberflächen',
```
Comment (ph1p) [inline gatsby-config.js]:
```suggestion
    title: 'React: Eine JavaScript Bibliothek zum Erstellen von Benutzeroberflächen',
```

File: src/components/TitleAndMetaTags/TitleAndMetaTags.js [hunk 3]
```diff
@@ -8,7 +8,7 @@
 import Helmet from 'react-helmet';
 import React from 'react';
 
-const defaultDescription = 'Eine JavaScript Bibliothek für das Bauen von Benutzeroberflächen';
+const defaultDescription = 'Eine JavaScript Bibliothek für das Erstellen von Benutzeroberflächen';
```
Comment (ph1p) [inline src/components/TitleAndMetaTags/TitleAndMetaTags.js]:
```suggestion
const defaultDescription = 'Eine JavaScript Bibliothek zum Erstellen von Benutzeroberflächen';
```

File: src/pages/index.js [hunk 4]
```diff
@@ -50,7 +50,7 @@ class Home extends Component {
     return (
       <Layout location={location}>
         <TitleAndMetaTags
-          title="React &ndash; Eine JavaScript Bibliothek für das Bauen von Benutzeroberflächen"
+          title="React &ndash; Eine JavaScript Bibliothek für das Erstellen von Benutzeroberflächen"
```
Comment (ph1p) [inline src/pages/index.js]:
```suggestion
          title="React &ndash; Eine JavaScript Bibliothek zum Erstellen von Benutzeroberflächen"
```

File: src/pages/index.js [hunk 5]
```diff
@@ -133,7 +133,7 @@ class Home extends Component {
                         fontSize: 30,
                       },
                     }}>
-                    Eine JavaScript Bibliothek für das Bauen von Benutzeroberflächen
+                    Eine JavaScript Bibliothek für das Erstellen von Benutzeroberflächen
```
Comment (ph1p) [inline src/pages/index.js]:
```suggestion
                    Eine JavaScript Bibliothek zum Erstellen von Benutzeroberflächen
```

**Pull request merged**